packages/headless/src/features/product-recommendations/product-recommendations-actions.ts

```diff
@@ -102,20 +102,35 @@ export const getProductRecommendations = createAsyncThunk<
   }
 );
 
-// TODO: Review the mappings here
 const mapResultToProductResult = (result: Result): ProductRecommendation => {
```

Comment:
This is a breaking change, but it's necessary since most of those fields are optional.

Those are based on the upcoming doc [here](https://docsstaging.coveo.com/COM-627-new-flow-of-catalog-creation/3139/coveo-for-commerce/creating-a-coveo-commerce-catalog#standard-fields-reference).